examples/coffeeshop.html

```diff
@@ -578,10 +578,27 @@ <H3 STYLE="flex-grow:1" ID="title">The CoffeeShop</H3>
       els.sample.contentDocument.body.style = "margin:0; padding:0;";
       postRun();
       els.sample.contentWindow.addEventListener('error', function(e) {
+        //console.log(e);
+        var rowOffset = 0;
+        if (e.filename == "") {
+          // Figure out line where `Algebra(..., ()=>{...}` starts in ACE Editor (because error is offsetted from there)
+          var lines = editor.getValue().split("\n");
+          rowOffset = -1;
+          for (var line of lines) {
+              if (line.match(/Algebra\s?\(/)) {
```

Comment:
Just a little bit more robust :)
```suggestion
              if (line.match(/\bAlgebra\s*\(/)) {
```

Reply:
Thanks a lot, fixed that in latest comment!